tensorboard/webapp/webapp_data_source/tb_http_client.ts

```diff
@@ -42,7 +64,24 @@ export class TBHttpClient implements TBHttpClientInterface {
     body: any,
     options: PostOptions = {}
   ): Observable<ResponseType> {
-    return this.http.post<ResponseType>(path, body, options);
+    // Google-internal Colab does not support HTTP POST requests, so we fall
```

Comment:
I think this comment would be better placed immediately under the line if(isInColab).  It does not accurately describe the entire code statement that follows it.

Reply:
Done.